### PR TITLE
Sizing fixes

### PR DESCRIPTION
**Pull request by schlosser**
A bunch of small visual tweaks:
- Text in weekview is slightly smaller
- Search box is only 75% width, and shares similar styling to the search box
- Search items hovers to blue
- Search items are not all bold anymore
- Move to consistent blues across stylesheets
- New "No courses" stylesheet that is actually a link that if clicked focuses on the searchbox.  Also has different text to be more descriptive of what this area is

screen shot 2014-07-30 at 2 39 00 am
screen shot 2014-07-30 at 2 37 50 am

### CODE REVIEW

File: server/startup/defaultSettings.js [hunk 1]
```diff
@@ -3,9 +3,14 @@ var defaultSettings = {
     host: 'localhost:9200'
   },
   facebook: {
-    appId: '837193842965373',
+    appId: '478856265465801',
```
Comment (natebrennand) [inline server/startup/defaultSettings.js:6]: why are we changing this?

Comment (schlosser) [inline server/startup/defaultSettings.js:6]: I didn't edit that file, possibly generated?  Or @brishin changed it and I included it somehow

**Comment by natebrennand** [comment]
let's get a confirmation from @brishin what's up with the FB ID. otherwise lgtm

**Comment by brishin** [comment]
It seems like it's happening because Dan got my diff as a a parent at some point, but the sha changed as it was merged into master. I'll fix it and merge.

**Comment by brishin** [comment]
@danrschlosser in the future, you can `git rebase -i master` and remove stray diffs and squash your own.

**Comment by brishin** [comment]
Just pushed these changes up. They look pretty awesome!

**Pull request merged**